EvidenceApi.Tests/V1/Gateways/EvidenceGatewayTests.cs

```diff
@@ -185,42 +186,71 @@ public void FindReturnsNullWhenADocumentSubmissionWithIdCannotBeFound()
+        {
+            var resident = TestDataHelper.Resident();
+            DatabaseContext.Residents.Add(resident);
+            DatabaseContext.SaveChanges();
+
+            var request = new EvidenceRequestsSearchQuery()
+            {
+                ServiceRequestedBy = "development-team-staging",
+                ResidentId = resident.Id,
+                State = EvidenceRequestState.Approved
+            };
+            var expected = ExpectedEvidenceRequests(resident, EvidenceRequestState.Approved);
+
+            var result = _classUnderTest.GetEvidenceRequests(request);
+            result.Should().BeEquivalentTo(expected);
+            request.ResidentId.Should().NotBeEmpty();
+        }
+
         [Test]
         public void CanGetEvidenceRequestsByServiceAndResidentId()
         {
             var resident = TestDataHelper.Resident();
             DatabaseContext.Residents.Add(resident);
 
             var expected = ExpectedEvidenceRequests(resident);
-            var serviceRequestedBy = "development-team-staging";
-            var residentId = resident.Id;
+            var request = new EvidenceRequestsSearchQuery()
+            {
+                ServiceRequestedBy = "development-team-staging",
+                ResidentId = resident.Id
+            };
 
-            var result = _classUnderTest.GetEvidenceRequests(serviceRequestedBy, residentId);
+            var result = _classUnderTest.GetEvidenceRequests(request);
             result.Should().BeEquivalentTo(expected);
-            residentId.Should().NotBeEmpty();
+            request.ResidentId.Should().NotBeEmpty();
         }
 
         [Test]
         public void CanGetEvidenceRequestsByServiceOnly()
         {
             var expected = ExpectedEvidenceRequests();
-            var serviceRequestedBy = "development-team-staging";
+            var request = new EvidenceRequestsSearchQuery()
+            {
+                ServiceRequestedBy = "development-team-staging"
+            };
 
-            var result = _classUnderTest.GetEvidenceRequests(serviceRequestedBy);
+            var result = _classUnderTest.GetEvidenceRequests(request);
             result.Should().BeEquivalentTo(expected);
         }
 
         [Test]
         public void GetEvidenceRequestsReturnsEmptyList()
         {
             var expected = ExpectedEvidenceRequests();
-            var serviceRequestedBy = "invalid-service";
+            var request = new EvidenceRequestsSearchQuery()
+            {
+                ServiceRequestedBy = "invalid-service"
+            };
 
-            var result = _classUnderTest.GetEvidenceRequests(serviceRequestedBy);
+            var result = _classUnderTest.GetEvidenceRequests(request);
             result.Should().BeEmpty();
         }
 
-        public List<EvidenceRequest> ExpectedEvidenceRequests(Resident? resident = null)
+        public List<EvidenceRequest> ExpectedEvidenceRequests(Resident? resident = null, EvidenceRequestState? state = null)
```

Comment:
Not quite sure I understand what this method is doing? It adds all the same attributes to both evidence requests, so how is it testing the filtering functionality